src/models/src/blocks/view.rs

```diff
@@ -71,5 +71,13 @@ pub struct SlackStatefulStateParams {
 #[skip_serializing_none]
 #[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
 pub struct SlackViewState {
-    pub values: HashMap<String, serde_json::Value>,
+    pub values: HashMap<SlackBlockId, HashMap<SlackActionId, SlackViewStateValue>>,
+}
+
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
+pub struct SlackViewStateValue {
+    #[serde(rename = "type")]
+    pub action_type: SlackActionType,
+    pub value: Option<String>,
```

Comment:
They in docs gave this example:
```
        "multiline": {
          "mlvalue": {
            "type": "plain_text_input",
            "value": "This is my example inputted value"
          }
        },
        "target_channel": {
          "target_select": {
            "type": "conversations_select",
            "selected_conversation": "C123B12DE"
          }
        }
``` 

As you can see they have different fields for different `action_type`:
`selected_conversation` instead of `value`. I think we should handle this because before this change people were able to read at least those if needed and now it would be unavailable.

Reply:
Wow, crazy typing of Slack API :(
I got a full list of field names such as `selected_conversation` from https://github.com/slackapi/bolt-js/blob/7eec4389253ad5b353af8d3cf0a15552eb18f4f1/src/types/view/index.ts#L132-L145 and working on it!

Reply:
I added them into SlackViewStateValue!
Respecting the bolt-js type definition, adding fields is better approach instead of changing it to an enum.

Reply:
@siketyan 
> Wow, crazy typing of Slack API :(

Yes, seems Slack team is really into dynamically typed languages internally and they don't care too much about proper types in some places.